vm/adb/adb.go

```diff
@@ -493,6 +495,8 @@ func (inst *instance) Copy(hostSrc string) (string, error) {
 	if _, err := inst.adb("push", hostSrc, vmDst); err != nil {
 		return "", err
 	}
+	inst.adb("shell", "chmod", "+x", "/data/syz-fuzzer")
```

Comment:
Let's make it more universal -- do chmod of the exact permissions that the file at `hostSrc` had.

Reply:
done